Fix 'Launch System' tutorial on Windows (#1416)
ros2/ros2_documentation: source/Tutorials/Launch-system.rst

Writing the launch file
^^^^^^^^^^^^^^^^^^^^^^^

> Inside your launch directory, create a new launch file with the ``_launch.py`` suffix.
> For example ``my_script_launch.py``.
> ``_launch.py`` is not specifically required as the file suffix for launch files.

Reviewer: I'd redo this whole thing as:

```suggestion
Inside your launch directory, create a new launch file called ``my_script_launch.py``.
``_launch.py`` is recommended, but not required, as the file suffix for launch files.
```

Writing the launch file
^^^^^^^^^^^^^^^^^^^^^^^

Inside your launch directory, create a new launch file with the ``_launch.py`` suffix.
For example ``my_script_launch.py``.

``_launch.py`` is not specifically required as the file suffix for launch files.
Another popular option is ``_launch.py``, used in the :ref:`beginner level launch files tutorial <ROS2Launch>`.

> If you do change the suffix, make sure to adjust the ``glob()`` argument in your ``setup.py`` file accordingly.

Reviewer: If we change the default to `_launch.py`, then the following 2 sentences don't make much sense.  Given that this is a beginner tutorial, I'd rather suggest just one "recommended" way to do it.  With that in mind, I'd just remove this paragraph entirely.

Author: Removed the paragraph in https://github.com/ros2/ros2_documentation/pull/1416/commits/55de33cf04439121961acdbcf3cc67a6d2a0b9e3.
I kept the sentence that says that `_launch.py` is not a required suffix.

.. code-block:: bash

   ros2 launch my_package my_script_launch.py

> Or for an standalone launch file:

Reviewer: ```suggestion
Or for a standalone launch file:
```
